deploy/examples/operator-openshift.yaml

```diff
@@ -584,6 +592,15 @@ data:
   # Defaults to all the labels mentioned in
   # https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/#osd-topology
   # CSI_CRUSH_LOCATION_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"
+
+  # -- specify below field to instruct Kubernetes to skip any attach operation altogether
+  # for PVC created by respective driver. More details at https://kubernetes-csi.github.io/docs/skip-attach.html#skip-attach-with-csi-driver-object
+  # If value is set to false it skips the volume attachments and makes the creation
+  # of pods using the PVC fast but it will have consequences of mounting RWO volumes on multiple nodes.
```

Comment:
Instead of saying it has consequences, should we say it's highly discouraged to use this for RWO volumes? And doesn't that mean the attacher likely should never be disabled for rbd, but commonly for cephfs? Some more specific recommendations like that would be helpful.

Reply:
Sure, i will reword it, if kubernetes properly enforces the RWO volumes to the single node, we can remove it. looks like its tracker in the kubernetes issue i mentioned but it might need some time.